### DISCUSSION TITLE
Is Marp could be fit the mobile devices?

### DISCUSSION THREAD

**Discussion by tianfanghan** [marp-team/marp#295] [Q&A]
Hi, I'v used Marp for VSCode for a long time to give presentation for my students on the web in html files. It's perfect, thank you.

There is a question, recently, more and more students started to use their mobile phone to view my slides on the class, although they can horizon their phone, but I would like to see a possibility to make my slides fit vertical view by phones, just like the picture. I borrow the picture from others sorry, but it shows my will.

![124759655-8c4dbe80-df38-11eb-9c17-d080d5b27430](https://user-images.githubusercontent.com/60310092/163495233-2e95de94-80b5-48ac-a15c-0528ea5a4a48.png)

**Comment by yhatt** [comment]
Each slides have a fixed size, and I think Marp is hard to make responsiveness to the viewport size.

To save the time of the slide author, Marp was created for that anyone do not have to think about [the responsive design](https://en.wikipedia.org/wiki/Responsive_web_design) of the slide. The fixed slide size just like a PowerPoint is a result of this thinking. If Marp had supported dynamic slide sizing to fit into the viewport, both of slide author and theme author must take a time to check a look in every possible viewports. Unfortunately, it's not our goal.

I think you have to try the slide tool that has no limit on the size of the slide viewport, e.g. [WebSlides](https://webslides.tv/). Fortunately, [Marpit framework](https://marpit.marp.app/) (the base framework of Marp) has [an example of WebSlides integration](https://codesandbox.io/s/j3wo2091yw), and you can try to write the slide with a feeling of Marp Markdown.

**Comment by tianfanghan** [comment]
Thank you very much. Yes, I know that Marp is to make us easy to make slides. Is the Marp WebSlides intergration can used in VSCode plugin? or I have to install Marpit from npm?
